tests/acceptance/build-results-test.js

```diff
@@ -120,7 +120,7 @@ test('displays appropriate indication for canary builds', function(assert) {
 });
 
 test('links to previous day', function(assert) {
-  let yesterday = moment().utc().subtract(1, 'day').format('YYYY-MM-DD');
+  let yesterday = moment().subtract(1, 'day').format('YYYY-MM-DD');
```

Comment:
Phil put a fix in for this here https://github.com/emberobserver/client/commit/91514d195fa3989786647b8e2f5d415a375dfd6d